lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -472,7 +632,7 @@ void SessionStateEnforcer::handle_state_update_to_amf(
   // Send message to AMF gRPC client handler.
   amf_srv_client_->handle_notification_to_access(notif);
 }
-bool SessionStateEnforcer::static_rule_init() {
+bool SessionStateEnforcer::default_and_static_rule_init() {

```

Comment:
This is a matter of preference but I prefer names like `initialize_default_and_static_rules`, just so that it is more readable

Reply:
Thanks for the suggestion, corrected.